tests/smartswitch/platform_tests/test_reload_dpu.py

```diff
@@ -17,18 +17,18 @@
 from tests.platform_tests.api.conftest import *  # noqa: F401,F403
 
 pytestmark = [
-    pytest.mark.topology('t1')
+    pytest.mark.topology('smartswitch-t1')
```

Comment:
the marker should be 'smartswitch', like pytest.mark.topology('t1') for all t1 topos and pytest.mark.topology('t0') for all t0.

Reply:
Changed it.